test/inductor/test_torchinductor.py

```diff
@@ -1460,6 +1460,87 @@ def test_conv_bn_fuse(self):
                         (v,),
                     )
 
+    # For gpu path, there has a accurcy issue,
+    @unittest.skipIf(HAS_CUDA, "only support cpu conv bn test")
+    def test_conv_customer_bn_fuse(self):
+        class CustomerBatchNorm(torch.nn.BatchNorm2d):
```

Comment:
The name "custom" is a bit confusing. It is actually conv+functional bn folding, right?

Reply:
yes, changed the name.